packages/paste-website/gatsby-config.js

```diff
@@ -4,11 +4,13 @@
  * See: https://www.gatsbyjs.org/docs/gatsby-config/
  */
 
-// const queries = require('./src/utils/algolia');
+const queries = require('./src/utils/algolia');
 
 require('dotenv').config();
 
-module.exports = {
+const gatsbyConfig = {};
+
+gatsbyConfig.data = {
```

Comment:
Not sure why we need this `data` key. Can we do
```
const gatsbyConfig = {
  siteMetadata: ...
}

if (proces.env.DEPLOYMENT_ENV === 'production') {
  gatsbyConfig.plugins.push({
    resolve: `gatsby-plugin-algolia`,
    options: {
      appId: process.env.GATSBY_ALGOLIA_APP_ID,
      apiKey: process.env.GATSBY_ALGOLIA_ADMIN_KEY,
      indexName: process.env.GATSBY_ALGOLIA_INDEX_NAME,
      queries,
      chunkSize: 10000,
    },
  });
}

module.exports = gatsbyConfig;
```

Reply:
Looks like I was just over-complicating things. Your solution is simpler and works.